src/main/java/com/library/service/impl/RabbitProducer.java

```diff
@@ -0,0 +1,80 @@
+package com.library.service.impl;
+
+import com.library.config.RabbitProperties;
+import com.library.constant.MessageType;
+import com.library.model.dto.LogMessage;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.AfterReturning;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.annotation.Pointcut;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static com.library.constant.ApplicationConstant.DATE_TIME_FORMATTER;
+
+@Aspect
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class RabbitProducer {
+
+    private final RabbitProperties rabbitProperties;
+
+    private final RabbitTemplate rabbitTemplate;
+
+    @Pointcut("execution(public * com.library.controller.BookController.*(..))")
+    public void callAtBookControllerPublic() { }
+
+    @Pointcut("execution(public * com.library.controller.DefaultExceptionHandler.*(..))")
+    public void callAtExceptionHandlerPublic() { }
+
+
+    @Before("callAtBookControllerPublic()")
+    public void sendRequiredArguments(JoinPoint joinPoint){
+        String s = Arrays.stream(joinPoint.getArgs()).map(Object::toString).collect(Collectors.joining(", "));
```

Comment:
Please, prepare example and show on call how it looks like in message.

Reply:
I can show it in screen(message in tg)